src/js/App.js

```diff
@@ -70,15 +76,15 @@ window.App = Backbone.View.extend({
     },
 
     slideOutStatsRegion: function() {
-        var $statsRegion = this.$statsRegion;
-        var model = this.model;
+        var self = this;
 
-        if ($statsRegion && this.mapView.$el) {
-            $statsRegion.addClass('slideout');
+        if (this.statsView.$el && this.mapView.$el) {
+            this.statsView.$el.addClass('slideout');
```

Comment:
It doesn't really get captured in a gif, but the stats region no longer slides out. It just disappears. This may require some tweaking of when the animation is kicked off and when the view is destroyed.

Reply:
Oof -- checking into this.

Reply:
This got broken on `develop` too so I'm going to make a new issue and handle fixing it there.